package.json

```diff
@@ -47,6 +48,7 @@
     "punycode": "^2.1.1",
     "react-async-hook": "^3.6.1",
     "react-hook-form": "5.1.3",
-    "valid-url": "^1.0.9"
+    "valid-url": "^1.0.9",
```

Comment:
Looks like you forgot to remove this package? At least it's not used in this PR.